spec/features/admin/submissions_spec.rb

```diff
@@ -47,7 +47,6 @@
             end
 
             it "successfully flags Submission" do
-              expect(page).to have_content("Response #{submission.id} was successfully flagged.")
```

Comment:
is there a way to assert the flag happened? maybe by flag color/status?

Reply:
Page having content Flagged asserts the flag happened